README-WINDOWS.md

```diff
@@ -0,0 +1,166 @@
+# Windows
+
+Let's get started!
+
+Ruby on Rails runs better in a *nix environment, so we'll use __VirtualBox__ and __Vagrant__
+to install __Linux__ on your machine.
+
+## Table of Contents
+
+- [Install VirtualBox](#install-virtualbox)
+- [Install Vagrant](#install-vagrant)
+- [Install Git](#install-git)
+  * [Adjusting your PATH environment](#adjusting-your-path-environment)
+  * [Configuring the line ending conversions](#configuring-the-line-ending-conversions)
+  * [Configuring the terminal emulator to use with Git Bash](#configuring-the-terminal-emulator-to-use-with-git-bash)
+- [Use a Pre-Built Linux (Vagrant) Box](#use-a-pre-built-linux-vagrant-box)
+- [Verify Everything Works!](#verify-everything-works)
+
+## Install [VirtualBox](https://www.virtualbox.org)
+
+[Download for Windows](https://www.virtualbox.org/wiki/Downloads),
+then run the Installer and follow the instructions.
+
+## Install [Vagrant](https://www.vagrantup.com)
+
+[Download for Windows](https://www.vagrantup.com/downloads.html),
+then run the Installer and follow the instructions.
+
+## Install Git
+
+Download [Git for Windows](https://git-scm.com/download/win).
+
+Run the installer, and choose the following options where appropriate (use default options for the rest):
+
+### Adjusting your PATH environment
+
+Choose option `Use Git and optional Unix tools from the Windows Command Prompt`.
+
+![Adjusting your PATH environment: Use Git and optional Unix tools from the Windows Command Prompt](images/git-001.png)
+
+### Configuring the line ending conversions
+
+Choose option `Checkout as-is, commit Unix-style line endings`.
+
+![Configuring the line ending conversions: Checkout as-is, commit Unix-style line endings](images/git-002.png)
+
+### Configuring the terminal emulator to use with Git Bash
+
+Choose option `Use Windows' default console window`.
+
+![Configuring the terminal emulator to use with Git Bash: Use Windows' default console window](images/git-003.png)
+
+## Use a Pre-Built _Linux_ (Vagrant) Box
+
+> _This should take about 10 min or less._ [See sample log](sample.log).
+
+Open up your Terminal.
+
+click on Windows Start and type "cmd" in the "Search programs and files" search field.
+
+Open `cmd.exe`. That's the Terminal equivalent in Windows.
+
+Create and change into into a directory which you'll be using for all of your coding needs, e.g. `~/workshop/` or `C:\workshop`:
```

Comment:
Create and change into into a directory which you'll be using for all of your coding needs, e.g. `C:\workshop`: